ObservatoryExplorer/DefaultCriteria.cs

```diff
@@ -145,7 +143,9 @@ public static List<(string Description, string Detail, bool SystemWide)> CheckIn
 
             if (settings.FastRotation && scan.RotationPeriod != 0 && !scan.TidalLock && Math.Abs(scan.RotationPeriod) < 28800 && !isRing)
             {
-                results.Add("Non-locked Body with Fast Rotation", $"Period: {scan.RotationPeriod/3600:N1} hours");
+                var spinningRemnants = new List<string> { "H", "N" };
+                if (!spinningRemnants.Contains(scan.StarType))
```

Comment:
Do we care about any fast-spinning star actually? (I'm pretty sure I've seen this trigger on some of the dwarf stars too.)

Reply:
Good point!